src/build/analyzer.ts

```diff
@@ -213,7 +215,7 @@ class StreamResolver implements Resolver {
       if (file) {
         deferred.resolve(file.contents.toString());
       } else {
-        console.log('No file found for', filepath);
+        logger.info('No file found for', filepath);
```

Comment:
regarding the log method APIs, I'm fine with only the first argument being the message, now that JS has string interpolation.